ocs_ci/framework/main.py

```diff
@@ -49,10 +50,17 @@ def init_ocsci_conf(arguments=None):
 
 def main(arguments):
     init_ocsci_conf(arguments)
+    run_id = int(time.time())
```

Comment:
Maybe move line 53 and 54 to init_ocsci_conf as it's part of initial config and probably doesn't have to be in main function directly.

For the rest of the functionality we can maybe thin of move it to some init_ocsci function which will be called after init_ocsci_conf and move there lines 55-58 and maybe also 65-67.

WDYT?

Reply:
First part is fixed by 6e0d644.

About the additional function `init_ocsci`, I'm not sure... If we will want to keep the `arguments.extend([...` part in the `main` function, we still need to define the `pytest_logs_dir`, which is basically the same how it is now... and if we will also move the `arguments.extend([...` part to the new function, there is not much what will left in the `main` function ... so not sure if it will bring more clarity or not...

Reply:
> First part is fixed by [6e0d644](https://github.com/red-hat-storage/ocs-ci/commit/6e0d644f1ce2d093d81cb592cce6b750b4f8afc3).
> 
> About the additional function `init_ocsci`, I'm not sure... If we will want to keep the `arguments.extend([...` part in the `main` function, we still need to define the `pytest_logs_dir`, which is basically the same how it is now... and if we will also move the `arguments.extend([...` part to the new function, there is not much what will left in the `main` function ... so not sure if it will bring more clarity or not...

I don't want to move arguments as they needs to be passed at the end, just lines I mentioned and if you will store this expanded path in the config, from main you can directly access from config and you don't need to expand users on multiple places if you will overwrite the config value.

In the feature we will need to add some other operation and I don't want to mess the main function here.